### PR TITLE
Rename source files and directories

### PR DESCRIPTION
**Pull request by jvanaalsburg**
This pull request organizes the source and test files into sub-modules and updates the imports. Splitting the levels and locations functionality into separate files (`levels.specified_levels`, `locations.group`, etc) is not included.

### CODE REVIEW

**Comment by adamkorynta** [comment]
Is the GitHub Actions failure expected?

File: cwms/timeseries/timeseries.py
Comment (adamkorynta) [inline cwms/timeseries/timeseries.py:1]: Any thoughts on this being `timeseries_numerical.py`? 

I'm thinking `timeseries.py` is fine for double data given how much more that type of data is usedw, but wanted to throw that out there.

Comment (adamkorynta) [reply cwms/timeseries/timeseries.py:1]: More cases against `timseries_numerical.py`:

CDA uses "timeseries" for numerical, all HEC/CWMS java objects use "timeseries", and the CWMS schema references numerical just as "ts".

**Comment by jvanaalsburg** [comment]
Thanks. I updated the module name and also fixed some imports that I missed in the first commit.

**Pull request merged**